src/components/Album.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import { ArtistName } from './ArtistName';
+import { CoverImage } from './CoverImage';
+import { AlbumName } from './AlbumName';
+
+export const Album = ({ album }) => {
+  return (
+    <div className="album-card">
+      <CoverImage imageUrl={album.images[0].url} />
+      <div className="album-info">
+        <div className="album-name">
+          <AlbumName name={album.name} url={album.external_urls.spotify} />
+        </div>
+        <div className="artist-name">
+          {album.artists.map((artist, index) => (
+            <ArtistName
+              key={artist.id}
+              name={artist.name}
+              url={artist.external_urls.spotify}
+              isLast={index === album.artists.length - 1}
+            />
+          ))}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+Album.propTypes = {
+  album: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    images: PropTypes.arrayOf(
+      PropTypes.shape({
+        url: PropTypes.string.isRequired,
+      })
+    ).isRequired,
+    external_urls: PropTypes.shape({
+      spotify: PropTypes.string.isRequired,
+    }).isRequired,
+    artists: PropTypes.arrayOf(
+      PropTypes.shape({
+        name: PropTypes.string.isRequired,
+        external_urls: PropTypes.shape({
+          spotify: PropTypes.string.isRequired,
+        }).isRequired,
+      })
+    ).isRequired,
+  }).isRequired,
+};
```

Comment:
Do you understand this part? We have not gone through it. It's important that you understand the code, even though you've gotten help from your collaborator. :)